### PR TITLE
Component indexed terms

### LINKED ISSUE #401: Display indexed terms on component detail page
Discovered while looking at #375 and #383. 

**Acceptance criteria**

Given a component that has indexed names (see #5) or subjects (see #11):

* Display an "Indexed Terms" heading like on the collection overview page
* Display groupings for names, subjects, and places below the heading, like on the collection overview page

### PR DESCRIPTION
**Pull request by gordonleacock**
Add indexed terms section on component show page

Closes #401 and #207

### CODE REVIEW

**Comment by anarchivist** [comment]
Hi @gordonleacock - is this PR ready to be reviewed? Which tickets does it close?

**Comment by gordonleacock** [comment]
@anarchivist - Issue “Display indexed terms on component detail page #401”
is ready to close by PR#424.

File: spec/features/indexing_custom_document_spec.rb [hunk 1]
```diff
@@ -74,7 +74,6 @@
 
     it '#level' do
       expect(doc['level_ssm'].first).to eq 'collection'
-      expect(doc['level_sim'].first).to eq 'Collection'
```
Comment (drh-stanford) [inline spec/features/indexing_custom_document_spec.rb]: i think these tests are all still valid.

File: spec/features/indexing_custom_document_spec.rb [hunk 2]
```diff
@@ -92,14 +91,12 @@
 
       it 'matches EAD data' do
         expect(doc['repository_ssm'].first).to eq '1118 Badger Vine Special Collections'
-        expect(doc['repository_sim'].first).to eq '1118 Badger Vine Special Collections'
```
Comment (drh-stanford) [inline spec/features/indexing_custom_document_spec.rb]: i think these tests are all still valid.

File: spec/features/indexing_custom_document_spec.rb [hunk 3]
```diff
@@ -92,14 +91,12 @@
...
       end
 
       context 'with REPOSITORY_ID' do
         it 'matches Repository configuration' do
           ENV['REPOSITORY_ID'] = 'nlm'
           expect(doc['repository_ssm'].first).to eq 'National Library of Medicine. History of Medicine Division'
-          expect(doc['repository_sim'].first).to eq 'National Library of Medicine. History of Medicine Division'
```
Comment (drh-stanford) [inline spec/features/indexing_custom_document_spec.rb]: i think these tests are all still valid.

**Comment by drh-stanford** [comment]
screen shot 2017-06-01 at 12 13 07 pm

File: spec/features/indexing_custom_document_spec.rb (hunk 2 above)
Comment (jkeck) [inline spec/features/indexing_custom_document_spec.rb]: I was curious about this too.  It looks like the expectation is an exact duplicate of the one above, or is there something I'm not seeing?

Comment (mejackreed) [inline spec/features/indexing_custom_document_spec.rb]: `ssm` vs `sim`

Comment (jkeck) [inline spec/features/indexing_custom_document_spec.rb]: `/facepalm`

Comment (mejackreed) [inline spec/features/indexing_custom_document_spec.rb]: It took me a minute too!

**Comment by drh-stanford** [comment]
@gordonleacock the PR build is failing due to the missing `id` parameter that's now required for `to_solr` calls in the specs. See https://github.com/sul-dlss/arclight/commit/f9059191b19f216fb175417ff60174510e807ba8 for the change in `indexing_custom_component_spec.rb`

**Pull request merged**